### ISSUE TITLE
Emoji overlay ignores bot messages (!hype)

### ISSUE THREAD

**Issue by mholloway24** [MichaelJolley/number-one#24]
**Describe the bug**
When the !hype command is invoked, the bot sends an emoji-filled message to chat which is NOT picked up by the emoji overlay

**To Reproduce**
1. type !hype <any_user>
2. B3_Bot sends hype emojis
3. Emojis are not seen on screen

**Expected behavior**
I expect the emojis to be visible on the screen when the bot sends the emoji messages for !hype

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser IE Edge
 - Version 85.0.564.44 (Official build) (64-bit)

**Additional context**

**Comment by MichaelJolley** [comment]
@all-contributors please add @mholloway24 for bugs

**Comment by MichaelJolley** [comment]
This repo doesn't have an overlay to display emotes like Vulcan.  Also, right now, all messages from the streamer and the bot are disregarded. 

I'd like to rethink how we handle emotes. Rather than randomly dancing around the screen, I'd like to do something else.  I'll create a new task for that so we can discuss what should happen.

Once that issue is done, maybe we alter the !hype command to not send a message to chat with emotes, but to trigger whatever event would fire an the event coded in the preceding task.

**Comment by MichaelJolley** [comment]
@allcontributors please add @mholloway24 for bugs
